Source/WebCore/platform/graphics/gstreamer/TrackPrivateBaseGStreamer.h

```diff
@@ -78,6 +80,11 @@ class TrackPrivateBaseGStreamer {
 
     virtual void tagsChanged(const GRefPtr<GstTagList>&) { }
     virtual void capsChanged(const String&, const GRefPtr<GstCaps>&) { }
+    void installUpdateConfigurationHandlers();
+    virtual void updateConfigurationFromCaps(const GRefPtr<GstCaps>&) { }
+    virtual void updateConfigurationFromTags(const GRefPtr<GstTagList>&) { }
```

Comment:
I'm seeing that, at least in this patch, you seem to be using these methods by passing its parameters as r-values. You could either change the signature of it to receive &&. If they are needed elsewhere as copy and the compiler can't handle the copy automatically, you can provide overloads that do the copy for you.